README.md

```diff
@@ -15,13 +15,6 @@ DuckDuckGo Privacy Extensions is distributed under the Apache 2.0 [License](LICE
 
 [Edge](https://microsoftedge.microsoft.com/addons/detail/duckduckgo-privacy-essent/caoacbimdbbljakfhgikoodekdnlcgpk)
 
-### Safari
```

Comment:
this tells people where to find the actual Safari extension, I think we should keep it